app/assets/stylesheets/index.scss

```diff
@@ -61,3 +78,30 @@ a {
 .pagination {
   text-align: center;
 }
+
+
+.footer {
+  background-color: #292c2f;
```

Comment:
Color literals like `#292c2f` should only be used in variable declarations; they should be referred to via variable everywhere else.